pcs/specs.py

```diff
@@ -63,6 +63,16 @@ def type(self) -> str:
     def as_kwargs(self):
         return {k: v for k, v in self.body.items() if k != "type"}
 
+    def to_dict(self, flatten=False):
```

Comment:
This class inherits from UserDict so it "is a" dict already. Does it still make sense to have a to_dict() method?

Reply:
Yeah good point.  This doesn't make much sense.   I did this to satisfy the DeepDiff type complaint issue.  We can dump this once we figure out the root cause of that.